src/effects/effectmanifest.h

```diff
@@ -82,6 +83,14 @@ class EffectManifest final {
         m_isMixingEQ = value;
     }
 
+    virtual const bool& isWaveformChangingEQ() const {
+        return m_isWaveformChangingEQ;
```

Comment:
It is probably more universal usable if this is a per parameter option. Did you think about that?

If we find a broad consensus to ditch this fake feature. It's also a solution.